reference-architecture/aws-ansible/playbooks/validation.yaml

```diff
@@ -6,10 +6,10 @@
   pre_tasks:
       - name: set fact
         set_fact:
-           openshift_master_cluster_public_hostname: "{{ openshift_master_cluster_public_hostname }}"
+           openshift_master_cluster_public_hostname: "openshift-master.{{ public_hosted_zone }}"
       - name: set fact
         set_fact:
-           openshift_master_cluster_hostname: "{{ openshift_master_cluster_hostname }}"
+           openshift_master_cluster_hostname: "internal-openshift-master.{{ public_hosted_zone }}"
```

Comment:
Is there a private hosted zone that could be used instead of the public_hosted_zone? This should only be applied to and used for the internal-only load balancer